.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
```

Comment:
Can prob drop this if using GH actions :)

Reply:
I think Travis needs this line, because when removed, it runs tests for Ruby.

Reply:
I will disable it on the new repo :)